sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -667,6 +667,15 @@ case class UnresolvedCatalogRelation(
   override def output: Seq[Attribute] = Nil
 }
 
+/**
+ * A wrapper to store the temporary view info, will be kept in `SessionCatalog`
+ * and will be transformed to `View` in `getTempView`
```

Comment:
`in getTempView`, there are more places, maybe just say `will be transformed to View during analysis`